eclair-core/src/test/scala/fr/acinq/eclair/payment/relay/NodeRelayerSpec.scala

```diff
@@ -723,26 +852,32 @@ object NodeRelayerSpec {
 
   val incomingAmount = 5000000 msat
   val incomingSecret = randomBytes32()
-  val incomingMultiPart = Seq(
+  val incomingMultiPart: Seq[NodeRelayPacket] = Seq(
     createValidIncomingPacket(2000000 msat, incomingAmount, CltvExpiry(500000), outgoingAmount, outgoingExpiry),
     createValidIncomingPacket(2000000 msat, incomingAmount, CltvExpiry(499999), outgoingAmount, outgoingExpiry),
     createValidIncomingPacket(1000000 msat, incomingAmount, CltvExpiry(499999), outgoingAmount, outgoingExpiry)
   )
   val incomingSinglePart = createValidIncomingPacket(incomingAmount, incomingAmount, CltvExpiry(500000), outgoingAmount, outgoingExpiry)
+  val incomingAsyncPayment: Seq[NodeRelayPacket] = incomingMultiPart.map(p => p.copy(innerPayload = IntermediatePayload.NodeRelay.Standard.createNodeRelayForAsyncPayment(p.innerPayload.amountToForward, p.innerPayload.outgoingCltv, outgoingNodeId)))
 
   def createSuccessEvent(): PaymentSent =
     PaymentSent(relayId, paymentHash, paymentPreimage, outgoingAmount, outgoingNodeId, Seq(PaymentSent.PartialPayment(UUID.randomUUID(), outgoingAmount, 10 msat, randomBytes32(), None)))
 
-  def createValidIncomingPacket(amountIn: MilliSatoshi, totalAmountIn: MilliSatoshi, expiryIn: CltvExpiry, amountOut: MilliSatoshi, expiryOut: CltvExpiry): IncomingPaymentPacket.NodeRelayPacket = {
+  def createValidIncomingPacket(amountIn: MilliSatoshi, totalAmountIn: MilliSatoshi, expiryIn: CltvExpiry, amountOut: MilliSatoshi, expiryOut: CltvExpiry, isAsyncPayment: Boolean = false): IncomingPaymentPacket.NodeRelayPacket = {
```

Comment:
It looks like you're not using `isAsyncPayment` anywhere...You should probably either revert the changes to that method or use it when creating your `incomingAsyncPayment` above (instead of modifying the `incomingMultiPart` onces), whatever you prefer!

Reply:
Removed it in [fcdc394](https://github.com/ACINQ/eclair/pull/2435/commits/fcdc394cb2751d7408318afb437cf49d18389d08)